conda/recipes/libcudf/conda_build_config.yaml

```diff
@@ -25,8 +25,8 @@ gtest_version:
 aws_sdk_cpp_version:
   - "<1.11"
 
-libarrow_version:
-  - "=12"
+libarrow:
```

Comment:
Was it intentional to drop the `_version` suffix?

Reply:
No 🙁 good catch. But now that it's done and we have CI passing, I'm inclined to merge and fix in a follow-up to unblock CI.